README.md

```diff
@@ -101,6 +105,7 @@ class Command(BaseCommand):
         logger.info(
             "Added weekly job: 'delete_old_job_executions'."
         )
+        register_events(scheduler)
```

Comment:
Calling `register_events` is no longer necessary.

Reply:
...to elaborate on this further, the jobstore will automatically register for the event that it cares about when it is [started](https://github.com/jarekwg/django-apscheduler/blob/master/django_apscheduler/jobstores.py#L35).

Reply:
But recently I try to run the demo `runspscheduler.py` without this line, all the job executions are not logged in the Django admin page. After I add this line, the logs appear in the Django admin page. I think this line might be the reason.

Reply:
[`register_events`](https://github.com/jarekwg/django-apscheduler/blob/58822559b3e83d9793e9747bbefc97c2bfcd6d0c/django_apscheduler/jobstores.py#L272-L279) is now a no-op that just raises a deprecation warning - not sure that it can affect anything else.